src/test/java/at/ac/tuwien/kr/alpha/config/CommandLineParserTest.java

```diff
@@ -28,13 +28,19 @@
 package at.ac.tuwien.kr.alpha.config;
 
 import org.apache.commons.cli.ParseException;
-import org.junit.Assert;
 import org.junit.Test;
 
 import java.util.Arrays;
+import java.util.function.Consumer;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 public class CommandLineParserTest {
 
+	private static final String DEFAULT_COMMAND_LINE = "java -jar Alpha-bundled.jar";
+	private static final Consumer<String> DEFAULT_ABORT_ACTION = (msg) -> { };
```

Comment:
At least not in the current form of `Main.ALPHA_CALL_SYNTAX`, because `ALPHA_CALL_SYNTAX = "java -jar alpha-bundled.jar" + System.lineSeparator() + "java -jar alpha.jar";`

Reply:
Since this is only the unit-test I don't think it makes any difference. Anyway, the value is only used by ```CommandLineParser``` to generate the usage message and doesn't influence command line parsing in any way. The current value of ```Main.ALPHA_CALL_SYNTAX``` reflects the two "normal" ways to call the jar file I could think of at the time. If one wanted to over-engineer this we'd read a property file there that is generated during the build and holds the actual name of the jar file being built